rotkehlchen/serialization/deserialize.py

```diff
@@ -533,6 +535,17 @@ def deserialize_evm_transaction(
     ...
 
 
+@overload
+def deserialize_evm_transaction(  # type: ignore[misc]
+        data: dict[str, Any],
+        internal: Literal[False],
+        chain_id: ChainID,
```

Comment:
This should also be specified

```suggestion
        chain_id: Literal[ChainID.OPTIMISM],
```